<?xml version='1.0' ?>

<launch>
  <node pkg="free_fleet_client_ros2" exec="fake_action_server" name="fake_action_server" />

  <node pkg="free_fleet_client_ros2" exec="fake_docking_server" name="fake_docking_server" />

  <node pkg="tf2_ros" exec="static_transform_publisher" name="fake_robot_transform" args="0.0 0.0 0.0 0.0 0.0 0.0 1.0 base_footprint map" output="both"/>

  <node pkg="free_fleet_client_ros2" exec="free_fleet_client_ros2" name="fake_client_node" output="both">
    <param name="fleet_name" value="fake_fleet"/>
    <param name="robot_name" value="fake_robot"/>
    <param name="robot_model" value="fake_robot_model"/>
    <param name="level_name" value="L1"/>
    <param name="dds_domain" value="42"/>
    <param name="max_dist_to_first_waypoint" value="10.0"/>
    <param name="nav2_server_name" value="/navigate_to_pose_fake"/>
    <param name="docking_trigger_server_name" value="/dock_fake"/>
  </node>

</launch>
